src/spacel/model/app.py

```diff
@@ -21,9 +21,17 @@ def __init__(self, orbit, params=None):
         self.instance_type = params.get('instance_type', 't2.nano')
         self.instance_min = params.get('instance_min', 1)
         self.instance_max = params.get('instance_max', 2)
-        self.scheme = params.get('scheme', 'internet-facing')
         self.health_check = params.get('health_check', 'TCP:80')
         self.local_health_check = params.get('health_check', 'TCP:80')
+        self.loadbalancer = self._str2bool(params.get('loadbalancer', 'true'))
```

Comment:
Can this just be `self.elb_availability == 'disabled'`? (instead of another flag).